src/GitExtensions.PluginTemplate/Plugin.cs

```diff
@@ -1,4 +1,5 @@
-﻿using GitExtensions.PluginTemplate.Properties;
+﻿using System.Collections.Generic;

```

Comment:
Why is this using directive needed?

Reply:
Link to the commented code that the developer could uncomment.